components/HomepageSlider/HomepageSlider.module.scss

```diff
@@ -0,0 +1,53 @@
+.slider {
+    top: 50%;
+    left: 50%;
+    transform: translate(0);
+    width: 588px;
+    height: 248px;
+    .slides {
+        transition: left .75s;
+        display: flex;
+        position: relative;
+        top: 0;
+        left: -588px;
```

Comment:
Just general question: I'm not too familiar with code practises in this project, but does this cover different screens and responsiveness?

Reply:
I think the answer to your question is: yes.

To elaborate: The slider will resize on mobile and desktop screens to properly fit. It will also re-adjust the slides properly when the slider size (responsive) changes.

If you look down in this scss file, you can see that we set data states for responsive screen sizes. It is basically the same as media queries, but if we want to have new screen sizes or adjust existing ones, we can adjust this now very easily globally in the project, rather than adjusting every single media query on its own.